ReCaptchaFrontendUi/view/frontend/web/js/reCaptcha.js

```diff
@@ -28,7 +28,17 @@ define(
              */
             initialize: function () {
                 this._super();
-                this._loadApi();
+
+                this.attachFocusEvent();
+            },
+
+            attachFocusEvent: function () {
+                let parentForm = $('#' + this.getReCaptchaId() + '-container').parents('form');
```

Comment:
1. This value never changes, so `const` is a better choice here than `let`.
2. Convention says that jQuery objects should be prefixed with a `$` symbol.
```suggestion
                const $parentForm = $('#' + this.getReCaptchaId() + '-container').parents('form');
```